modules/livestore/live_store.go

```diff
@@ -123,6 +123,7 @@ type LiveStore struct {
 	wg              sync.WaitGroup
 	completeQueues  *flushqueues.ExclusiveQueues
 	startupComplete chan struct{} // channel to signal that the starting function has finished. allows background processes to block until the service is fully started
+	lagCancel       context.CancelFunc
```

Comment:
The partition lag cant use the normal context and needs to be explicitly closed first before stopping else logs leak into tests.